src/sxastarter/src/components/play-travel/Accordion.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionPanel,
+  AccordionIcon,
+  Box,
+} from '@chakra-ui/react';
+
+import {
+  ComponentParams,
+  ComponentRendering,
+  Text as JssTextField,
+  Image as JssImageField,
+  TextField,
+  RichTextField,
+  ImageField,
+} from '@sitecore-jss/sitecore-jss-nextjs';
+
+interface AccordionItemProps {
+  heading: { jsonValue: TextField };
+  content: { jsonValue: RichTextField };
+  image: { jsonValue: ImageField };
+}
+
+interface Fields {
+  data: {
+    accordion: {
+      children: {
+        results: AccordionItemProps[];
+      };
+    };
+  };
+}
+
+interface AccordionProps {
+  rendering: ComponentRendering & { params: ComponentParams };
+  params: ComponentParams;
+  fields: Fields;
+}
+
+const AccordionItemRendering = (props: AccordionItemProps) => (
+  <AccordionItem>
+    <h2>
+      <AccordionButton
+        className={'h-20'}
+        _hover={{
+          bgColor: 'white',
+        }}
+      >
+        <Box className={'flex justify-between font-bold w-full'} fontSize={'28px'}>
+          <JssTextField field={props.heading.jsonValue} />
+          <AccordionIcon color={'#515151'} />
+        </Box>
+      </AccordionButton>
+    </h2>
+    <AccordionPanel
+      borderTopWidth={`1px`}
+      paddingBottom={{ base: '20px', md: '28px' }}
+      paddingRight={{ base: 0, md: '12px' }}
+      paddingLeft={{ base: 0, md: '24px' }}
+      paddingTop={{ base: 0, md: '18px' }}
+      className={'flex justify-between flex-col md:flex-row'}
+    >
+      <Box
+        className={'min-w-full max-w-full mt-5 md:mt-2.5 md:mr-10 mb-6 md:mb-0'}
+        minWidth={{ md: '123px' }}
+        maxWidth={{ md: '123px' }}
+        minHeight={{ base: '123px' }}
+        maxHeight={{ base: '123px' }}
+        height={{ base: '123px' }}
+      >
+        <JssImageField
+          style={{ width: '100%', height: '100%' }}
+          alt={props?.heading.jsonValue.value}
+          field={props?.image.jsonValue}
+          className={'w-full object-cover'}
+        />
+      </Box>
+      <JssTextField
+        field={props.content.jsonValue}
+        className={'text-base'}
+        style={{ lineHeight: '29px', color: '#6D7079' }}
```

Comment:
if you use this way of specifying lineHeight, color, it will work in Editing mode. Please check these props for the Preview mode. They don't appear on the page and text color if black instead of being gray